.github/workflows/snyk-infrastructure.yml

```diff
@@ -3,20 +3,18 @@
 # separate terms of service, privacy policy, and support
 # documentation.
 
-# A sample workflow which checks out the code, builds a container
-# image using Docker and scans that image for vulnerabilities using
-# Snyk. The results are then uploaded to GitHub Security Code Scanning
+# A sample workflow which checks out your Infrastructure as Code Configuration files,
+# such as Kubernetes, Helm & Terraform and scans them for any security issues.
+# The results are then uploaded to GitHub Security Code Scanning
 #
-# For more examples, including how to limit scans to only high-severity
-# issues, monitor images for newly disclosed vulnerabilities in Snyk and
-# fail PR checks for new vulnerabilities, see https://github.com/snyk/actions/
+# For more examples, including how to limit scans to only high-severity issues
+# and fail PR checks, see https://github.com/snyk/actions/
 
-name: Snyk Container
+name: Snyk Infrastructure as Code
 
 on:
   schedule:
-    - cron: '30 22 * * 1,4' # At 22:30 on Monday and Thursday.
-  workflow_dispatch:
```

Comment:
Might want to keep `workflow_dispatch:` so we can run this manually if need be.

Reply:
Good shout